README.md

```diff
@@ -2029,6 +2031,52 @@ case class MyDao(c: MyContext) extends MySchema {
 }
 ```
 
+### Extended Contexts
+
+Another simple way to extend contexts is by extending `Context` as a self-type and using mixins. Using this strategy,
+it is possible to create functionality that is fully portable across databases and even types of databases
+(e.g. creating common queries for both Postgres and Spark).
+
+For example, take the following abstract context:
+
+```
+import io.getquill.idiom.Idiom
+import io.getquill.context.Context
+
+trait MultiDbContext[I <: Idiom, N <: NamingStrategy] { this: Context[I, N] =>
+  def peopleOlderThen = quote {
```

Comment:
Specially because it's documentation: `peopleOlderThan`

Reply:
Fixed